src/config/config_impl.go

```diff
@@ -252,6 +267,28 @@ func (this *rateLimitConfigImpl) GetLimit(
 		return rateLimit
 	}
 
+	if descriptor.GetLimit() != nil {
+		rateLimitKey := domain + "." + this.descriptorToKey(descriptor)
+		rateLimit := value.overrides[rateLimitKey]
```

Comment:
Is this caching for perf? If so can you add more comments?

Reply:
Dropped it